seed/java-sdk/imdb/disable-required-property-builder-checks/src/main/java/com/seed/api/resources/imdb/ImdbClient.java

```diff
@@ -3,30 +3,29 @@
  */
 package com.seed.api.resources.imdb;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.seed.api.core.ClientOptions;
-import com.seed.api.core.MediaTypes;
-import com.seed.api.core.ObjectMappers;
 import com.seed.api.core.RequestOptions;
-import com.seed.api.core.SeedApiApiException;
-import com.seed.api.core.SeedApiException;
-import com.seed.api.resources.imdb.errors.MovieDoesNotExistError;
+import com.seed.api.core.Suppliers;
+import com.seed.api.resources.imdb.http.ImdbHttpClient;
 import com.seed.api.resources.imdb.types.CreateMovieRequest;
 import com.seed.api.resources.imdb.types.Movie;
-import java.io.IOException;
-import okhttp3.Headers;
-import okhttp3.HttpUrl;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.RequestBody;
-import okhttp3.Response;
-import okhttp3.ResponseBody;
+import java.util.function.Supplier;
 
 public class ImdbClient {
     protected final ClientOptions clientOptions;
 
+    protected final Supplier<ImdbHttpClient> httpClient;
```

Comment:
Thoughts on `RawImdbClient` and `AsyncRawImdbClient` (i.e. where `Raw` is the prefix to `Imdb`)?